app/models/user.rb

```diff
@@ -45,6 +45,10 @@ class User < ApplicationRecord
   acts_as_followable
   acts_as_follower
 
+  has_many :authored_user_subscriptions, class_name: "UserSubscription", foreign_key: :author_id, inverse_of: :author, dependent: :destroy
```

Comment:
Yes, I think the passive voice does actually clear it up a bit more. 

BTW: recent discussion on `foreign_key`: https://github.com/thepracticaldev/dev.to/pull/8685#discussion_r440304148